bin/RegisterTritonDialects.h

```diff
@@ -42,6 +47,28 @@ inline void registerTritonDialects(mlir::DialectRegistry &registry) {
   mlir::triton::registerConvertNVGPUToLLVMPass();
   mlir::registerLLVMDIScope();
 
+#ifdef USE_ROCM
+  mlir::triton::registerConvertTritonAMDGPUToLLVM();
+
+  // TODO: Uncomment when fixed undefined symbols and
```

Comment:
Do you know the root cause of this?

Reply:
If I remember correctly, these passes are defined in tablegen and in some headers, but not implemented anywhere at the moment.

So to fix this we need to either remove definition or implement AMD specific (maybe copy Nvidia pass first).